CoinDock-app/src/App/Api/coincardapi.js

```diff
@@ -0,0 +1,56 @@
+import { getUserId } from "App/Auth/helper";
+import baseApi from "./api";
+baseApi.enhanceEndpoints({
+  addTagTypes: ["coincard"],
+});
+const coincardapi = baseApi.injectEndpoints({
+  endpoints: (build) => ({
+    coincard: build.query({
+      query: () => ({
+        // url: `/v1/users/${getUserId()}/recovery-codes`,
+        url: `/v1/users/${getUserId()}/coin-cards/`,
+        method: "get",
+      }),
+
+      // transformResponse: (response) => {
```

Comment:
Can you remove comments please if not needed

Reply:
removed